semver/__init__.py

```diff
@@ -1067,15 +1073,21 @@ def test_set(set_, version,  include_prerelease=False):
 def satisfies(version, range_, loose=False, include_prerelease=False):
     try:
         range_ = make_range(range_, loose)
-    except Exception as e:
+    except InvalidTypeIncluded:
```

Comment:
This exception is subclass of ValueError, handling specially (TypeError is good if this is first implementation, but historical reason, passing invalid value is treated as ValueError, so this error is subclass of ValueError) .